python/lsst/ts/watcher/rules/electric_generator_fail.py

```diff
@@ -0,0 +1,187 @@
+    def get_schema(cls):
+        indent = " " * 8
+        severity_values = "\n".join(
+            [f"{indent}- {severity.name}" for severity in AlarmSeverity]
+        )
+        schema_yaml = f"""
+$schema: http://json-schema.org/draft-07/schema#
+description: >-
+    Configuration for electric generators failure monitoring.
+type: object
+properties:
+    name_master:
+        description: >-
+            CSC name and index in the form `name:index`
+            for the master electric generator.
+        type: string
+    name_slave:
+        description: >-
+            CSC name and index in the form `name:index`
+            for the slave electric generator.
+        type: string
+    severity_individual_fail:
+        description: >-
+            Alarm severity for when one electric generator has
+            a main failure.
+        type: string
+        default: {AlarmSeverity.SERIOUS.name}
+        enum:
+{severity_values}
+    severity_both_fail:
+        description: >-
+            Alarm severity for when both electric generators
+            have a main failure.
+        type: string
+        default: {AlarmSeverity.CRITICAL.name}
+        enum:
+{severity_values}
+required:
+  - name_master
+  - name_slave
+additionalProperties: false
+       """
+        return yaml.safe_load(schema_yaml)
+
+    def compute_alarm_severity(
+        self, data: salobj.BaseMsgType, **kwargs: typing.Any
+    ) -> AlarmSeverityReasonType:
+        """Compute and set alarm severity and reason.
+
+        Parameters
+        ----------
+        data : `salobj.BaseMsgType`
+              Message from the topic described by topic_callback.
+        **kwargs : `dict` [`str`, `typing.Any`]
+            Keyword arguments. If triggered by `TopicCallback` calling
+            `update_alarm_severity`, the arguments will be as follows:
+
+            * topic_callback : `TopicCallback`
+              Topic callback wrapper.
+
+        Returns
+        -------
+        None, if no change or unknown, or a tuple of two values:
+
+        severity: `lsst.ts.xml.enums.Watcher.AlarmSeverity`
+            The new alarm severity.
+        reason : `str`
+            Detailed reason for the severity, e.g. a string describing
+            what value is out of range, and what the range is.
+            If ``severity`` is ``NONE`` then this value is ignored (but still
+            required) and the old reason is retained until the alarm is reset
+            to ``nominal`` state.
+
+        Notes
+        -----
+        You may return `NoneNoReason` if the alarm state is ``NONE``.
+        """
+        remote_name_master, remote_index_master = salobj.name_to_name_index(
+            self.config.name_master
+        )
+        remote_name_slave, remote_index_slave = salobj.name_to_name_index(
+            self.config.name_slave
+        )
+        severity, reason = NoneNoReason
+        genset_failure = getattr(data, "mainFailure")
+        if genset_failure is True:
```

Comment:
Why use the `is` terminology here? See guidelines [here](https://developer.lsst.io/python/style.html#is-and-is-not-should-only-be-used-for-determining-if-two-variables-point-to-same-object) on use of `is`.

Reply:
Had doubts on this one too, thanks for the heads up!